CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 - Add `enableCodeCoverage` generation option to enable code coverage in automatically generated schemes [#2020](https://github.com/tuist/tuist/pull/2020) by [@frijole](https://github.com/frijole).)
 - Add support for Command Line Tool targets [#1941](https://github.com/tuist/tuist/pull/1941) by [@olejnjak](https://github.com/olejnjak).
 
+### Fixed
+
+- Fixed signing linter for target with bundle identifier derived from build settings [#2031] by [@leszko11](https://github.com/leszko11).
```

Comment:
this will now appear at the wrong place once merged -> please rebase/merge with `main` and put it into `# Next` section